src/WebJobs.Extensions.DurableTask/Bindings/OrchestrationTriggerAttributeBindingProvider.cs

```diff
@@ -53,6 +53,7 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
 
             // The orchestration name defaults to the method name.
             var orchestratorName = new FunctionName(name, trigger.Version);
+            this.config.RegisterActivity(orchestratorName, null);
```

Comment:
This needs to be a call to `RegisterOrchestrator`, which will also need the same new logic that you added for `RegisterActivity`.

Reply:
Thank you for catching that! I will fix it. Thank you!